util/syspolicy/policy_keys.go

```diff
@@ -170,6 +174,7 @@ var implicitDefinitions = []*setting.Definition{
 	setting.NewDefinition(ManagedByURL, setting.UserSetting, setting.StringValue),
 	setting.NewDefinition(NetworkDevicesVisibility, setting.UserSetting, setting.VisibilityValue),
 	setting.NewDefinition(PreferencesMenuVisibility, setting.UserSetting, setting.VisibilityValue),
+	setting.NewDefinition(RequireReasonToDisconnect, setting.DeviceSetting, setting.BooleanValue),
```

Comment:
```suggestion
	setting.NewDefinition(RequireReasonToDisconnect, setting.UserSetting, setting.BooleanValue),
```

It is listed among other policy settings that should be configurable on a per-user basis, which is good and makes perfect sense. However, the scope should be `setting.UserSetting` for it to work correctly.

Reply:
This should line up with what's in Kabir's proposal here... Or Kabir's proposal should be edited to line up with this.

https://docs.google.com/document/d/1Rji0zM8NfzrBf-bY9umHxEFlKhQuOcWeXUa2EdijpYQ/edit?tab=t.0#heading=h.21np41gfymly

"ForceEnabledAndAllowDisconnectWithReason"

Reply:
Done, good catch.